order_analysis.rb

```diff
@@ -3,17 +3,20 @@
 
 class OrderAnalysis
   CSV_DELIMITER = ";"
-
+  def initialize
+    @customers = []
+  end
   def analyze
     analyze_customers
+    analyze_orders
   end
-
   def analyze_customers
```

Comment:
Method name is misleading, as it doesn‘t do any analyzation, but parses csv file.

```suggestion
  def parse_customer_csv
```

Reply:
haha true, but that part was already like that in the original file from the ruby monstas: [https://github.com/rubymonstas-zurich/order-analysis-exercise/blob/main/order_analysis.rb](url)